### ISSUE TITLE
The Mandela Effect Scanning my Memory Lists

### ISSUE THREAD

**Issue by sethrice** [egzumer/uv-k5-firmware-custom#631]
Does this sound correct? If i'm scanning a list, the radio won't get any RX on the other VFO on screen. Only on the VFO where the scan is cycling through. Which means even if I have a frequency from that list, static on the top VFO. It won't pick up any traffic, until the bottom VFO with the scan running, gets to that frequency?

I'm trying to determine if I tweaked a config, or if I just didn't recognize/understand until now how the scanning function works.  Now the Monopoly Man is looking at me confused and all like, "That's... not my monocle."

Let's say I have my radio setup with two channels that i'm listening to, top and bottom VFO as..
 Channel 001 = 151.115 - List I
 **Channel 052** = 463.180  - List II

My understanding of the scanning function, when using scanning lists. Was that you could have a static frequency or memory channel set on the top VFO. The scan setup on that bottom VFO, cycling through list 2. If a transmission comes in on Channel 001, then you hear right away what comes in on that channel, on the frequency set. In addition the scan keeps going, on that bottom VFO. If by some chance, not in this example, but perhaps another. If that channel was in the same scan list, it would just stop at that channel. 

However what I discovered today, was that I am totally off base on this. 

The reality I've discovered, is i'm scanning through a scan list, then the frequency chilling on the VFO not being scanned, will never receive traffic. As if I go from dual band RX radio, to just single band, totally held by the scanning list. If that top VFO happens to be included in the scan list, then the scan will stop. It might even appear as though it started receiving on the top VFO, and the bottom stops scanning when it catches up and reaches that same frequency.

However it appears that's not the case. It also doesn't seem to matter if i'm scanning on the top VFO, or the bottom. 

I tested this out by...

- First putting my two Quansheng UV-K6 side by side, connected to the same antenna with a splitter.
- I then grabbed an Arcshell radio. Set that to channel 8, then set the top VFO on both Quansheng radios to that channel 8 frequency, by doing the frequency copy function.
- Tested to confirm I could receive on both Quansheng radios. I then added all arcshell frequencies to a scan list on both radios. 
- I started the scan on one radio, but left the other radio so that it was set to a different frequency all together on that bottom VFO. Both radios have the top VFO set to that Arcshell channel 8 frequency. 

When attempting a test broadcast from arcshell, I was able to receive on the Quansheng radio instantly that had the two static frequencies set. However the radio that had the arcshell frequency set on the top VFO, and the scan rolling through the arcshell frequency list on the bottom VFO, didn't pick up the transmission right away. Not until the Arcshell channel 8 frequency came up in the scan list cycle. This didn't seem to matter which frequency/channel was on the top or bottom VFO. 

Basically, if the scanner is scanning a list, you don't get any RX on the other VFO on screen. You can't scan a list and monitor a static frequency at the same time. Like how you can monitor two static frequencies at the same time?

Does that sound right?

🤦 = Cool Story Bro, you've discovered something you should have already understood about scanners. Go get your Amateur Radio license, and stay off the internet until you do.... slacker.

🤔 = You might be on to something. Now go get your ARL, and stop procrastinating.

**Comment by prokrypt** [comment]
Working as intended. If you want the other VFO channel to be monitored while scanning, I have that feature enabled in my fork, called "Voxless"...
